.circleci/config.yml

```diff
@@ -527,7 +529,7 @@ jobs:
       - run:
           name: Check sdist distribution
           command: |
-            THISVERSION=$( python -m hatch version )
+            THISVERSION=$( python -m hatch version | tail -n1 )
             THISVERSION=${THISVERSION%.d*}
```

Comment:
Oh, it's probably this.

Reply:
Yup, though now I'm wondering why we had this substitution in the first place

Reply:
It used to be intended to remove the `.dirty` tag from versioneer.